internal/ansible/controller/reconcile.go

```diff
@@ -255,7 +255,7 @@ func (r *AnsibleOperatorReconciler) Reconcile(ctx context.Context, request recon
 		}
 	} else if recentlyDeleted && finalizerExists {
 		// If the CR was deleted after the reconcile began, we need to requeue for the finalizer.
-		reconcileResult.Requeue = true
+		reconcileResult.RequeueAfter = 5 * time.Second
```

Comment:
It is deprecated, seems a good change 👍 


https://github.com/kubernetes-sigs/controller-runtime/blob/b9a9ca01fd37a66225eb4793544349ee2ed1f196/pkg/reconcile/reconcile.go#L31-L43